tests/integration/test_integration.py

```diff
@@ -54,3 +55,27 @@ async def test_given_default_config_when_deploy_then_status_is_active(
         await ops_test.model.wait_for_idle(  # type: ignore[union-attr]
             apps=[APPLICATION_NAME], status="active", timeout=1000
         )
+
+    @pytest.mark.abort_on_fail
+    async def test_given_application_is_deployed_when_scale_up_then_status_is_active(
+        self,
+        ops_test: OpsTest,
+        build_and_deploy,
+    ):
+        await ops_test.model.applications[APPLICATION_NAME].scale(7)  # type: ignore[union-attr]
+
+        await ops_test.model.wait_for_idle(  # type: ignore[union-attr]
+            apps=[APPLICATION_NAME], status="active", timeout=1000
+        )
```

Comment:
You will also need to add the parameter `wait_for_exact_units` to the number you are expecting, otherwise this will always pass because one unit is already `active/idle`.